### DISCUSSION TITLE
拉起钱包切换链

### DISCUSSION THREAD

**Discussion by Amazing123511** [WalletConnect/walletconnect-monorepo#4272] [Developer Support] [answered]
我使用walletconnect2.0 的Ethereum Provider 接入,我想拉起钱包切换到指定链,拉不起来

**Comment by glitch-txs** [comment, marked as answer]
Hi, you need to use this method: https://eips.ethereum.org/EIPS/eip-3326 and fallback on this one: https://eips.ethereum.org/EIPS/eip-3085
